sql-cli/sql_cli/settings.py

```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
+
+from airflow.configuration import conf
+
+PROJECT_BASE_PATH = Path(__file__).parent.parent.resolve()
+
+# This needs to be updated with the `astro dev init` command to set the appropriate project directory.
```

Comment:
I believe we should not depend on `astro dev init`, only on `flow init` to set up a SQL project. I also believe we should not rely on an Airflow configuration to define the project root directory.

I can refactor and remove this as part of the flow init PR #829